### ISSUE TITLE
[no-private-routing-service] TypeError: Cannot read property 'value' of undefined

### ISSUE THREAD

**Issue by boris-petrov** [ember-cli/eslint-plugin-ember#696]
```
Occurred while linting /home/boris/code/project/some-file.ts:80
    at Property (/home/boris/code/project/node_modules/eslint-plugin-ember/lib/rules/no-private-routing-service.js:36:35)
    at /home/boris/code/project/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/home/boris/code/project/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/home/boris/code/project/node_modules/eslint/lib/linter/node-event-generator.js:254:26)
    at NodeEventGenerator.applySelectors (/home/boris/code/project/node_modules/eslint/lib/linter/node-event-generator.js:283:22)
    at NodeEventGenerator.enterNode (/home/boris/code/project/node_modules/eslint/lib/linter/node-event-generator.js:297:14)
    at CodePathAnalyzer.enterNode (/home/boris/code/project/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:634:23)
    at /home/boris/code/project/node_modules/eslint/lib/linter/linter.js:936:32
    at Array.forEach (<anonymous>)
```

That file around that line looks like:

```typescript
import Service, { inject } from '@ember/service';

Service.reopen(foo, {
  session: inject(),
  bar: readOnly('session.isLoggedIn'),
});
```

**Comment by bmish** [comment]
@nlfurniss want to take a look?

**Comment by nlfurniss** [comment]
Ooof, sorry about that. Let me fix it

**Comment by nlfurniss** [comment]
@bmish fixed in #697
